tensoradapter/pytorch/torch.cpp

```diff
@@ -47,6 +50,16 @@ TA_EXPORTS DLManagedTensor* TAempty(
   return at::toDLPack(tensor);
 }
 
+#ifdef DGL_USE_CUDA
+TA_EXPORTS void* RawAlloc(size_t nbytes) {
+  return c10::cuda::CUDACachingAllocator::raw_alloc(nbytes);
```

Comment:
Can we just use `torch::empty`?

Reply:
Then we have to change the behavior of `AllocWorkspace` because we have to save the tensor to reuse its memory. We either have to:
1. Change all codes using `AllocWorkspace`.
2. Save the allocated tensors somewhere and return `tensor.data_prt` as raw pointers. But when deleting, we have to search for the tensor that matches the pointer to be freed.

Reply:
You are right.